lib/adapter/freshheads/CopyFilesToBuildDirAdapter.ts

```diff
@@ -4,19 +4,25 @@ import { BuilderConfig } from '../../Builder';
 import path from 'path';
 import { validateIfRequiredModuleIsInstalled } from '../../utility/moduleHelper';
 
-// @todo use types from @types/copy-webpack-plugin instead (cannot export CopyPattern right now)
-type CopyPattern = {
-    from: {
-        glob: string;
-    };
-    context?: string;
+// @todo use types from @types/copy-webpack-plugin instead (cannot export StringPattern / ObjectPattern right now)
+type StringPattern = string;
+interface ObjectPattern {
+    from: string;
     to?: string;
+    context?: string;
+    globOptions?: object;
     toType?: 'file' | 'dir' | 'template';
-};
+    force?: boolean;
+    flatten?: boolean;
+    transform?: (content: Buffer, absoluteFrom: string) => string | Buffer | Promise<string | Buffer>;
+    cacheTransform?: boolean | string | object;
+    transformPath?: (targetPath: string, absolutePath: string) => string | Promise<string>;
+    noErrorOnMissing?: boolean;
+}
 
 export type Config = {
     images: boolean;
-    additionalPatterns: CopyPattern[];
+    additionalPatterns: Array<StringPattern | ObjectPattern>;
```

Comment:
Je dupliceert twee keer `Array<StringPattern | ObjectPattern>`. Misschien handig om zelf ook even in een type te zetten, zodat er geen foutgevoeligheid ontstaat bij veranderingen. (DRY)